backend/src/Types/Database.hs

```diff
@@ -334,3 +231,174 @@ updatedPlayerStatsLose side rating (PlayerStatsTotal {..}) (PlayerStatsYear {..}
           ..
         }
     )
+
+instance ToNamedRecord Player where
+  toNamedRecord :: Player -> CSV.NamedRecord
+  toNamedRecord Player {..} =
+    CSV.namedRecord
+      [ "name" .= playerName,
+        "display_name" .= playerDisplayName,
+        "country" .= fromMaybe ("" :: Text) playerCountry,
+        "is_active" .= if playerIsActive then ("Active" :: Text) else "Inactive"
```

Comment:
I like the readability of mapping this column from true/false to active/inactive 👍